projects/storefrontlib/src/cms-components/checkout/components/abstract-checkout-step/abstract-checkout-step.component.ts

```diff
@@ -0,0 +1,27 @@
+import { OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { CheckoutConfigService } from '../../services/checkout-config.service';
+
+export abstract class AbstractCheckoutStepComponent implements OnInit {
+  checkoutStepUrlNext: string;
+  checkoutStepUrlPrevious: string;
+
+  backButtonText = 'common.back';
```

Comment:
This should be a constant

Reply:
A class member cannot have 'const' keyword.